tauembedding_tagandprobe.py

```diff
@@ -411,146 +498,182 @@ def build_config(
+                            "max_deltaR_triggermatch": 0.1,
+                            "triggerobject_ptcut": -1,
+                        },
+                        {
+                            "flagname_1": "trg_Mu17TrkMu8_DZ_bit_12_Mu17_1",
+                            "flagname_2": "trg_Mu17TrkMu8_DZ_bit_12_Mu17_2",
+                            "ptcut": 17.0,
+                            "etacut": 2.5,
+                            "filterbit": 12,
                             "trigger_particle_id": 13,
                             "max_deltaR_triggermatch": 0.1,
                             "triggerobject_ptcut": 17.0,
                         },
                         {
-                            "flagname_1": "trg_Mu17TrkMu8_DZ_Mu8_1",
-                            "flagname_2": "trg_Mu17TrkMu8_DZ_Mu8_2",
-                            "hlt_path": "HLT_Mu17_TrkIsoVVL_Mu8_TrkIsoVVL_DZ",
-                            "ptcut": 8,
+                            "flagname_1": "trg_Mu17TrkMu8_DZ_bit_12_Mu8_1",
+                            "flagname_2": "trg_Mu17TrkMu8_DZ_bit_12_Mu8_2",
+                            "ptcut": 8.0,
                             "etacut": 2.5,
-                            "filterbit": -1,
+                            "filterbit": 12,
                             "trigger_particle_id": 13,
                             "max_deltaR_triggermatch": 0.1,
                             "triggerobject_ptcut": 8.0,
                         },
                         {
-                            "flagname_1": "trg_Mu17TrkMu8_DZMass8_Mu17_1",
-                            "flagname_2": "trg_Mu17TrkMu8_DZMass8_Mu17_2",
-                            "hlt_path": "HLT_Mu17_TrkIsoVVL_Mu8_TrkIsoVVL_DZ_Mass8",
-                            "ptcut": 17,
+                            "flagname_1": "trg_Mu17TrkMu8_DZ_bit_13_Mu17_1",
+                            "flagname_2": "trg_Mu17TrkMu8_DZ_bit_13_Mu17_2",
+                            "ptcut": 17.0,
                             "etacut": 2.5,
-                            "filterbit": -1,
+                            "filterbit": 13,
                             "trigger_particle_id": 13,
                             "max_deltaR_triggermatch": 0.1,
                             "triggerobject_ptcut": 17.0,
                         },
                         {
-                            "flagname_1": "trg_Mu17TrkMu8_DZMass8_Mu8_1",
-                            "flagname_2": "trg_Mu17TrkMu8_DZMass8_Mu8_2",
-                            "hlt_path": "HLT_Mu17_TrkIsoVVL_Mu8_TrkIsoVVL_DZ_Mass8",
-                            "ptcut": 8,
+                            "flagname_1": "trg_Mu17TrkMu8_DZ_bit_13_Mu8_1",
+                            "flagname_2": "trg_Mu17TrkMu8_DZ_bit_13_Mu8_2",
+                            "ptcut": 8.0,
                             "etacut": 2.5,
-                            "filterbit": -1,
+                            "filterbit": 13,
```

Comment:
For the creation of the custom NanoAOD (`CMSSW_10_6_35`) I directly modified `PhysicsTools/NanoAOD/python/triggerObjects_cff.py`. I left a hint in the corresponding comment block for this request, since the number might change depending on potential new necessities in the future. Where else should I document it?

Reply:
my preference would be a short markdown in https://github.com/KIT-CMS/TagAndProbe that contains all changes required for the measurement and can be used as a recipe to perform the measurement again. You could just update the `README.md` in `TagAndProbe` with the commands for the mesurement and document the nanoAOD change there in an additional section.